pynest/nest/lib/hl_api_connections.py

```diff
@@ -522,7 +475,7 @@ def CGParse(xml_filename):
     generator cg.
 
     The library to provide the parsing can be selected
-    by CGSelectImplementation().
+    by :py:func:`.CGSelectImplementation`.
```

Comment:
OK, if this link works, use this format for all links in previous .rst files (not sure the initial ``:py`` is actually necessary, is it?)